client/a8c-for-agencies/components/items-dashboard/items-dataviews/index.tsx

```diff
@@ -121,6 +121,7 @@ const ItemsDataViews = ( { data, isLoading = false, className }: ItemsDataViewsP
 						return item.id;
 					} )
 				}
+				onSelectionChange={ data.onSelectionChange }
```

Comment:
I don't see us adding an onSelectionChange on the dotcom side for this?

Reply:
There is an issue on the dotcom. See https://github.com/Automattic/wp-calypso/pull/91034#issuecomment-2128500688